front/src/modules/ui/table/editable-cell/type/components/GenericEditableNumberCellEditMode.tsx

```diff
@@ -31,12 +35,27 @@ export function GenericEditableNumberCellEditMode({ viewField }: OwnProps) {
     if (newText === fieldValue) return;
 
     try {
-      const numberValue = parseInt(newText);
+      let numberValue = parseInt(newText);
 
       if (isNaN(numberValue)) {
```

Comment:
We can remove this check as it is already done in canBeCastAsPositiveIntegerOrNull

Reply:
Hey @Weiko, thanks for the review.  canBeCastAsPositiveIntegerOrNull check will only happen when isPositive flag is true. isNaN check is outside of that guard. If isPositive flag is false, we can't use canBeCastAsPositiveIntegerOrNull at all. Please correct me if I'm wrong. Thanks

Reply:
That's true 🙂. I think at some point we can refactor both utils (castAsInteger and the new castAsPositive), the logic is very similar. The *AsInteger should do most of the validations and be called before *AsPositiveInteger imo. We can think about it later 👍